DEVELOPMENT.md

```diff
@@ -0,0 +1,70 @@
+# Development Uploadcare Widget
+
+Clone the repository.
```

Comment:
Clone the repository. Now.

`Make sure you're making changes in your local copy of our repo.`?